Detectors/MUON/MCH/Evaluation/README.md

```diff
@@ -17,3 +17,40 @@ $ o2-mch-tracks-reader-workflow --hbfutils-config o2_tfidinfo.root --infile mcht
 $ root
 root[0] o2::mch::eval::drawAll("compare.root")
 ```
+# MCH Cluster Maps
+
+The general purpose is to track "unexpected" detector issues not well reproduced with MC simulations. These problems generate non-negligible bias in Acc*Eff corrections resulting in large tracking systematic uncertainties.
+
+During the data reconstruction, the status of the detector is calculated with the CCDB which is used to discard most of the detector issues.
+This status map is built with information based on pedestals, high voltage tension, occupancy etc.
+
+Nevertheless, some detector issues (e.g. a cable swapping) are not well detected online and consequently not properly reproduced by the CCBD.
+The main objective of this code is to spot these issues not included in the status map.
+
+Input files used:
```

Comment:
I guess the documentation on how to use this program should be primarily seen from the point of the view of the next person that will _use it_ to produce cluster map comparisons, not so much develop the code. So how to get the input files should be part of the documentation for instance. (by the way that doc could be put in a separate `.md` file to keep the main README.md more as an index.